modules/sor/balancer-sor.integration.test.ts

```diff
@@ -329,6 +329,79 @@ describe('Balancer SOR Integration Tests', () => {
         });
     });
 
+    describe('Buffer Pool Path', () => {
+        beforeAll(async () => {
+            // setup mock pool data
+            const poolAddress = '0x302b75a27e5e157f93c679dd7a25fdfcdbc1473c';
+            const stataUSDC = prismaPoolTokenFactory.build({
+                address: '0x8a88124522dbbf1e56352ba3de1d9f78c143751e',
+                token: { decimals: 6, underlyingTokenAddress: '0x94a9d9ac8a22534e3faca9f4e7f2e2cf85d5e4c8' },
+                dynamicData: prismaPoolTokenDynamicDataFactory.build({
+                    balance: '500',
```

Comment:
Do you use Tenderly to get the pool state? Might be worth adding a bit of documentation at the top of the file explaining the process so its easier for the next person?

Reply:
Yeah, this was a pretty manual process. I'll add a bit of documentation to get this PR ready, but I'll open another issue aiming to streamline this testData process. Ideally with something similar to Balancer Maths repo.

Reply:
Done ✅ 
I hope to find some time to improve this on the next few weeks. It's quite annoying to update manually all that test data.
On top of that, subgraph prunes old blocks, so we're not even able to verify the data if needed 😕